Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml

```diff
@@ -1177,6 +1223,516 @@ script:
+        auto: PREDEFINED
+        predefined:
+          - and
+          - or
+        required: true
+        isArray: true
+      - name: receiver_domain_entries
+        description: Comma-separated receiver domain entries (e.g., user@example.com, @example.com) used to match messages sent to these recipients.
+        isArray: true
+        required: true
+      - name: receiver_not_domain_entries
+        description: Comma-separated receiver domain entries to exclude. Used to match messages not sent to these recipients. Only allowed when `receiver_operation` is set to and.
+        isArray: true
+  - name: cisco-esa-incoming-policy-user-update
+    description: Update sender and recipient entries in an existing incoming mail policy. This command overrides all existing sender and recipient entries with the provided values.
+    arguments:
+      - name: mode
+        description: Defines the configuration level for applying changes on the email gateway. Set to 'cluster' to update the entire cluster, `group` to target a specific group (requires `group_name`), or 'machine' to update a single machine (requires `host_name`).
+        defaultValue: cluster
+        auto: PREDEFINED
+        predefined:
+          - cluster
+          - group
+          - machine
+      - name: group_name
+        description: The name of the group to target when 'mode' is set to `group`. This value is required only if mode is `group`.
+      - name: host_name
+        description: The hostname of the machine to target when 'mode' is set to 'machine'. This value is required only if mode is 'machine'.
+      - name: policy_name
+        description: The name of the policy to edit all the sender and recipient entries.
+        default: true
+        required: true
+      - name: sender_domain_entries
+        description: Comma-separated sender domain entries (e.g., user@example.com, @example.com) used to match messages from these senders. Required only if `sender_non_domain_entries` is not provided. Cannot be used together with `sender_non_domain_entries`.
+        isArray: true
+      - name: sender_non_domain_entries
+        description: Comma-separated sender domain entries to exclude. Used to match messages not from these senders. Required only if `sender_domain_entries` is not provided. Cannot be used together with `sender_domain_entries`.
+        isArray: true
+      - name: receiver_operation
+        description: Specifies how to combine receiver matching conditions. Use `and` if `receiver_not_domain_entries` is provided. Use `or` if using ANY as a domain.
+        auto: PREDEFINED
+        predefined:
+          - and
+          - or
+        required: true
+        isArray: true
+      - name: receiver_domain_entries
+        description: Comma-separated receiver domain entries (e.g., user@example.com, @example.com) used to match messages sent to these recipients.
```

Comment:
Comma-separated recipient domain entries (for example, user@example.com, @example.com) used to match messages sent to these recipients.